pkg/scheduler/objects/application_test.go

```diff
@@ -1343,6 +1353,9 @@ func TestReplaceAllocationTracking(t *testing.T) {
 	assert.NilError(t, err, "could not add ask")
 	app.addPlaceholderDataWithLocking(ph3.GetAsk())
 
+	// set time to 1 second, in order to test placeholder resource usage
+	time.Sleep(1 * time.Second)
```

Comment:
Why do we have to sleep? I haven't seen this before in the tracking unit tests.

Reply:
Thank you, @pbacsko, for review, we sleep for those cases want to count the resource:

sleepTime * resourceUsageType, so we added those sleep time, and the value is resourceType * seconds, but we also may find a way to mock it, let me look into it.

And if we can't mock it, i think we one second is the most time we want to sleep, 3 seconds is too long.

Reply:
@pbacsko  addressed your comments and also fix all the time sleep for tracking resources.